src/router/router.ts

```diff
@@ -23,4 +23,4 @@ export function getVueRouter () {
   })
 }
 
-export { appBaseURL }
\ No newline at end of file
+export { appBaseURL }
```

Comment:
are these all the changes you wanted to have?

Reply:
yes